### PR TITLE
add LastInfoSec enrichment

### PR DESCRIPTION
**Pull request by remydewa**
### Proposed changes

* This pull request add the LastInfoSec enricher to OpenCTI to enrich domains and hashes.

### Related issues

* N/A

### Checklist

- [X] This connector enrich domain and hash indicators by calling the LastInfoSec api.
- [X] I consider the submitted work as finished
- [X] I tested the code for its functionality using different use cases
- [X] I added/update the relevant documentation (either on github or on notion)

### Further comments

### CODE REVIEW

File: internal-enrichment/lastinfosec/src/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pycti==4.5.5
```
Comment (nor3th) [inline internal-enrichment/lastinfosec/src/requirements.txt]: Please use the 5.0.2 pycti library for the connector (yes there's a version discrepancy between OpenCTI and the connectors (currently 5.0.1) and the pycti lib (currently 5.0.2).

**Comment by nor3th** [comment]
Also please run `black .` to adapt the format of your code. 
Details: https://www.notion.so/Connector-Development-06b2690697404b5ebc6e3556a1385940#265b804393f64922aeff232315a6a171

**Comment by remydewa** [comment]
Thanks nor3th! I did it. It looks good now :).

File: internal-enrichment/lastinfosec/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  connector-lastinfosec-enrichment:
+    image: opencti/lastinfosec-enrichment:4.5.5
```
Comment (nor3th) [inline internal-enrichment/lastinfosec/docker-compose.yml]: Please also change this to 5.0.1

**Review by nor3th — CHANGES_REQUESTED** [review]
hey @remydewa 

Thank you very much for you effort.

Additionally to the comments, please make use of the PR template for your submission, as this helps us to get a better understanding of your work.

(This comment should have already been posted earlier, but I forgot to submit the review...)

Thanks

**Comment by remydewa** [comment]
hello @nor3th 
Thanks, I did the changes to the merge request content explaination.
Rémy

**Comment by nor3th** [comment]
Great, but also please fix the mentioned version issues.

Thanks

**Comment by remydewa** [comment]
Hello @nor3th 

It's done. I hope this time everything is ok?
Thanks,
Rémy

**Comment by nor3th** [comment]
Hey @remydewa 

Yes, thank you very much for your effort! 

Regards,

**Pull request merged**